package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1255,24 +1260,26 @@ def moment_of_inertia(group, pbc=False, **kwargs):
         ('moment_of_inertia', moment_of_inertia))
 
     @warn_if_not_unique
-    def radius_of_gyration(group, pbc=False, **kwargs):
+    @_pbc_to_wrap
+    def radius_of_gyration(group, wrap=False, **kwargs):
         """Radius of gyration.
 
         Parameters
         ----------
-        pbc : bool, optional
+        wrap : bool, optional
```

Comment:
Could you please document the `unwrap` and `compound` parameters that were added in 0.20.0? I know, it's not your fault. But at the moment the docs are confusing, especially as both are not even explicit (and hidden as kwarg).  Making them explicit in the signature would also be good.

Reply:
Not applicable. Ignore above comment.